lib/gvariant.nix

```diff
@@ -0,0 +1,266 @@
+# This file is based on https://github.com/nix-community/home-manager
+# Copyright (c) 2017-2022 Home Manager contributors
+#
+# A partial and basic implementation of GVariant formatted strings. See https://docs.gtk.org/glib/gvariant-format-strings.html
+# for detauls.
+#
+# Note, this API is not considered fully stable and it might therefore
+# change in backwards incompatible ways without prior notice.
```

Comment:
This should probably be included in the docs as well.

Reply:
And maybe we should express it also in the attribute path, see also `pkgs._experimental-update-script-combinators`. Not completely sure about this.

Reply:
Where should I add it? Is it enough to convert it to a `/**/` style comment?

Reply:
Looks like nixdoc currently does not support module docs. Will try to implement it.

Reply:
Do we need to wait for that?